lib/hammer_cli_foreman_tasks/task.rb

```diff
@@ -12,6 +12,11 @@ def create_option_builder
     module ActionField
       def extend_data(task)
         task["action"] = [task["humanized"]["action"], task["humanized"]["input"]].join(' ')
+        task["duration"] = if task["started_at"] && task["ended_at"]
+                            Time.parse(task["ended_at"]).to_f - Time.parse(task["started_at"]).to_f
+                          elsif task["started_at"] && task["ended_at"].nil?
+                            Time.now.to_f - Time.parse(task["started_at"]).to_f
+                          end
```

Comment:
Instead of having to know how to calculate duration for a task in all the different places, couldn't we just expose the calculated duration in the api?

Reply:
Okay, I'll try to fix that